configure

```diff
@@ -937,22 +959,41 @@ def configure_node(o):
   else:
     o['variables']['node_enable_v8_vtunejit'] = 'false'
 
+  if flavor != 'linux' and (options.enable_pgo_generate or options.enable_pgo_use):
+    raise Exception(
+      'The pgo option is supported only on linux.')
+
+  if flavor == 'linux':
+    if options.enable_pgo_generate or options.enable_pgo_use:
+      version_checked = (5, 4, 1)
+      if check_gcc_version(version_checked) == False:
+        version_checked_str = ".".join(map(str, version_checked))
+        raise Exception(
+          'The options --enable-pgo-generate and --enable-pgo-use options '
+          'are supported for gcc and gxx %s or newer only.' % (version_checked_str))
+
+    if options.enable_pgo_generate and options.enable_pgo_use:
+      raise Exception(
+        'Only one of the --enable-pgo-generate or --enable-pgo-use options '
+        'can be specified at a time. You would like to use '
+        '--enable-pgo-generate first, profile node, and then recompile '
+        'with --enable-pgo-use')
+
+  o['variables']['enable_pgo_generate'] = b(options.enable_pgo_generate)
+  o['variables']['enable_pgo_use']      = b(options.enable_pgo_use)
+
   if flavor != 'linux' and (options.enable_lto):
     raise Exception(
       'The lto option is supported only on linux.')
 
   if flavor == 'linux':
     if options.enable_lto:
       version_checked = (5, 4, 1)
-      for compiler in [(CC, 'c'), (CXX, 'c++')]:
-        ok, is_clang, clang_version, compiler_version = \
-          try_check_compiler(compiler[0], compiler[1])
-        compiler_version_num = tuple(map(int, compiler_version))
-        if is_clang or compiler_version_num < version_checked:
-          version_checked_str = ".".join(map(str, version_checked))
-          raise Exception(
-            'The option --enable-lto is supported for gcc and gxx %s'
-            ' or newer only.' % (version_checked_str))
+      if check_gcc_version(version_checked) == False:
```

Comment:
`if not check_gcc_version(version_checked):`

Reply:
Agree and tried to resolve it.